kt-extensions/src/test/java/com/mindera/skeletoid/kt/extensions/utils/CalendarTest.kt

```diff
@@ -0,0 +1,50 @@
+package com.mindera.skeletoid.kt.extensions.utils
+
+import android.os.Build
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import java.text.SimpleDateFormat
+import java.util.Calendar
+
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = [Build.VERSION_CODES.O_MR1])
+class CalendarTest {
```

Comment:
Are we testing addDays extenstion here? Maybe it could have a better name instead of CalendarTest?

Reply:
I named the test class after the class is testing `Calendar.kt`